channels/slack/src/main/kotlin/com/justai/jaicf/channel/slack/SlackChannel.kt

```diff
@@ -38,10 +40,21 @@ class SlackChannel private constructor(
     private constructor(botApi: BotApi, urlPrefix: String): this(botApi) {
```

Comment:
@Denire What do you think about providing the constructor that accepts a complete Slack's `AppConfig`?

Reply:
Not sure. At least not now. 

I don't see any reasons to recongifure AppConfig as it works OK now with defaults, and as we use only OAuth authorization in our channel.